src/engine/comment.js

```diff
@@ -31,7 +31,7 @@ class Comment {
     toXML () {
         return `<comment id="${this.id}" x="${this.x}" y="${
             this.y}" w="${this.width}" h="${this.height}" pinned="${
-            this.blockId !== null}" collapsed="${this.minimized}">${xmlEscape(this.text)}</comment>`;
+            !this.minimized}" collapsed="${this.minimized}">${xmlEscape(this.text)}</comment>`;
```

Comment:
The `collapsed` attribute is kept for backwards compatibility?

Reply:
That's used by workspace comments.